spectator/log_entry.h

```diff
@@ -13,6 +13,7 @@ class LogEntry {
       : registry_{registry},
         start_{absl::Now()},
         id_{Id::Of("ipc.client.call", {{"nf.process", registry->GetConfig().process_name},
+                                       {"owner", registry->GetConfig().process_name},
```

Comment:
It should probably just be `spectatord`, as that is the name of the IPC integration. It isn't the same as the process name which can be more arbitrary.